src/extension/src/constants.ts

```diff
@@ -278,11 +278,13 @@ export const CONSTANTS = {
   },
   VALIDATION_LIMIT: 3,
   APP_SERVICE_DEPLOYMENT: {
+    SERVER_FOLDER: "server",
     DOT_VSCODE_FOLDER: ".vscode",
     SETTINGS_FILE_NAME: "settings.json",
-    SETTINGS_FILE: (id: string) => {
+    SETTINGS_FILE: (id: string, subpath: string) => {
       return `{
-    "appService.defaultWebAppToDeploy": "${id}"
+    "appService.defaultWebAppToDeploy": "${id}",
+    "appService.deploySubpath":"${subpath}"
```

Comment:
missing space after `:`? not running prettier?

Reply:
oops, thanks, I do have format on save, but this was not caught because it is a string